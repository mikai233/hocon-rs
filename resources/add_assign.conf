item1 = {
  item2 += {

  }

  item2 += {name = "hello"}
  item3 += {
    item5 += {
      name = "world"
      item6 += 1
      item7 += [1.0]
    }
    item5 += -1
  }
  item4 = ["hello"]
  item4 += {}


}