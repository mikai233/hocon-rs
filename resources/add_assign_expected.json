{
  "item1": {
    "item2": [
      {},
      {
        "name": "hello"
      }
    ],
    "item3": [
      {
        "item5": [
          {
            "name": "world",
            "item6": [
              1
            ],
            "item7": [
              [
                1.0
              ]
            ]
          },
          -1
        ]
      }
    ],
    "item4": [
      "hello",
      {}
    ]
  }
}